test/integration/logbucket-org/logbucket_org_test.go

```diff
@@ -50,6 +51,7 @@ func TestLogBucketOrgModule(t *testing.T) {
 		// assert log sink name, destination & filter
 		assert.Equal(logSinkDestination, logSinkDetails.Get("destination").String(), "log sink destination should match")
 		assert.Equal("resource.type = gce_instance", logSinkDetails.Get("filter").String(), "log sink filter should match")
+		assert.Contains(logSinkDetails.Get("writerIdentity").String(), logSinkWriterId, "log sink service identity should have the bucketWriter role")
```

Comment:
```suggestion
		assert.Contains(logSinkDetails.Get("writerIdentity").String(), logSinkWriterId, "log sink has expected identity")
```

Reply:
We should also do a check for the IAM role.

Reply:
Yes, you're right. I thought I did that already.